src/main/java/project_16x16/Camera.java

```diff
@@ -141,20 +141,20 @@ public void post() {
 		applet.stroke(0, 150, 255);
 		applet.strokeWeight(2);
 		final int length = 20;
-		applet.line(applet.width / 2 - length, applet.height / 2, applet.width / 2 + length, applet.height / 2);
-		applet.line(applet.width / 2, applet.height / 2 - length, applet.width / 2, applet.height / 2 + length);
+		applet.line(applet.width / 2.0f - length, applet.height / 2.0f, applet.width / 2.0f + length, applet.height / 2.0f);
```

Comment:
2.0f is here (and elsewhere) to force Java not to round the result (avoiding integer division)

Reply:
I forced floating point division here because I thought it was good coding practice, but now I realize that integer division is necessary here because we want integers in this case. I will revert these changes as well.

Reply:
You can actually re-revert this and put back the `2.0f`s. I got confused and thought you changed `2.0f` to `2`, not the other way around!